client/src/components/ReadMore.jsx

```diff
@@ -5,7 +5,7 @@ const ReadMore = (props) => (
     <h3>Getting Around</h3>
     <p>{props.gettingAround}</p>
     <h3>Guidebook</h3>
-    <p>For more information, checkout the guidebook.</p>
+    <p>For more information, checkout the {props.guidebook}.</p>
```

Comment:
you can destructure your props here if you want too - to remain consistent with your code in other places. You can do it directly within the function parameter on line 3 (ask me about that if you want to learn more about it) or you can add a return line and wrap your function in curly braces. lemme know if i can make that clearer if you want.